pom.xml

```diff
@@ -158,6 +170,11 @@
 						</excludes>
 					</configuration>
 				</plugin>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-surefire-plugin</artifactId>
+					<version>${maven-surefire-plugin.version}</version>
```

Comment:
Please move version in `pluginManagement`

Reply:
This section _is_ the pluginManagement.

Reply:
Oh, sorry, I got confused by the github diff.